src/main/java/com/google/sps/data/Vendor.java

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.Entity;
+
+/** A vendor. */
+public final class Vendor {
+
+  private final String id;
+  private String firstName;
+  private String lastName;
+  private String email;
+  private String phoneNumber;
+  private Picture profilePic;
+  private SaleCard businessInfo;
```

Comment:
```suggestion
  private SaleCard saleCard;
```
I think naming this saleCard will help the readability